axelrod/load_data_.py

```diff
@@ -0,0 +1,45 @@
+import pkg_resources
```

Comment:
I agree that this wouldn't test anything further, it just consolidates things: for example in the future these ml strategies could be changed to no longer read the data (hypothetically), their tests adjusted and an error creeping in to these reader functions. That's a weird case but I think my point holds?